deploy/static/provider/aws/deploy-tls-termination.yaml

```diff
@@ -38,10 +38,10 @@ metadata:
   namespace: ingress-nginx
 data:
   http-snippet: |
-    server {
-      listen 2443;
-      return 308 https://$host$request_uri;
-    }
+	server {
```

Comment:
Not needed here

Reply:
the problem here is a damn bug in generate-deploy-scripts.sh :/

Anyway, in this case this is a nit, @longwuyuan if you have time to fix the spacing here it's good, otherwise we can release the chart this way just to have this fixed :)

Reply:
OK, got it.

I used to manually restore the changes here

Reply:
yeah, me too. So I'm thinking about approving this to release the new chart, and we should fix in the script (Long is already working with the new contributors to fix this)